pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/Messages.properties

```diff
@@ -152,7 +152,7 @@ ModelValidatorImpl.MissingInputMessage=No message specified for input
 WhenConditionalValidator.changelog.missingParameter=Changelog is missing required parameter "pattern".
 WhenConditionalValidator.changelog.badPattern="{0}" is not a valid regular expression. {1}
 
-ModelInterpreter.NoNodeContext=Attempted to execute a step that requires a node context while \'agent none\' was specified. Be sure to specify your own \'node { ... }\' blocks when using \'agent none\'.
+ModelInterpreter.NoNodeContext=Attempted to execute a step that requires a node context while \u2018agent none\u2019 was specified. Be sure to specify your own \u2018node '{ ... }'\u2019 blocks when using \u2018agent none\u2019.
```

Comment:
I do not get it. What would

```groovy
node '{ ... }'
```

possibly mean? I think the `'`s should be deleted here just as they were in the other two cases in this line.

Reply:
@jglick: they're talking to `MessageFormatter.format`, and they tell it "This `{` and its contents including the `...` and the closing brace `}` are not to be treated like `{0}`"  -- something which it would try to substitute with an argument.